source/_patterns/01-molecules/27-styled-ibetype-icons/styled-ibetype-icons.mustache

```diff
@@ -0,0 +1,3 @@
+<span class="styled-ibetype-icons">
```

Comment:
styled-ibetype-icons -> styled-ibe-type-icons
The naming convention for CSS is a dash between words.

Reply:
Sure, I just considered "IBE Type" as only one word for some reason. I'll separate then, thanks.